MAUI/Kanban-Board/Cards.md

```diff
@@ -136,34 +136,34 @@ Create a custom class by inheriting [`DataTemplateSelector`](), and override the
 
 {% highlight c# %}
 
-    public class KanbanTemplateSelector : DataTemplateSelector
```

Comment:
@EzhilarasanElangovan31 at line 129, update DataTemplateSelector link - https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.datatemplateselector?view=net-maui-8.0

Reply:
Updated the link